README.md

```diff
@@ -9,6 +9,183 @@ Cerberus watches the Kubernetes/OpenShift clusters for dead nodes, system compon
 ![Cerberus workflow](media/cerberus-workflow.png)
 
 
+## Ways to install and use Cerberus
```

Comment:
@mffiedler this needs to be part the docs/installation.md file.

Reply:
ok, i still screwed up the merge.   Let me rework it.